### ISSUE TITLE
Actors not linked

### ISSUE THREAD

**Issue by jesshmusic** [Pariah-Zero/dead-in-thay#2]
When I try to open a scene it says none of the actors are linked.

<img width="927" alt="Screenshot 2024-03-22 at 3 26 16 PM" src="https://github.com/Pariah-Zero/dead-in-thay/assets/7180584/e10e298f-907f-4589-9560-d48b1ee334e7">

**Comment by Pariah-Zero** [comment]
Ok. I'll make sure I have this taken care of (along with initial v12 compatibility) in the next update. Hopefully this weekend.

**Comment by Pariah-Zero** [comment]
@jesshmusic : Quick question - is it _just_ the actor image?

If it is (as I suspect), it's probably a duplicate of #1 - for which there is [a workaround](https://github.com/Pariah-Zero/dead-in-thay/issues/1#issuecomment-1789844727).

I had really hoped to be done with this by now, but circumstances have not played nicely.

**Comment by jesshmusic** [comment]
No, when I try to double click an actor it says the token no longer references it
